README.md

```diff
@@ -199,7 +199,7 @@ type Decorator = (value: Input, context: {
 }) => Output | void;
 ```
 
-`Input` and `Output` here represent the values passed to and returned from a given decorator. Each type of decorator has a different input and output, and these are covered below in more detail. All decorators can choose to return nothing, which defaults to using the original, undecorated value.
+`Input` and `Output` here represent the values passed to and returned from a given decorator. Each type of decorator has a different input and output, and these are covered below in more detail. All decorators can choose to return nothing (undefined, or void in TypeScript), which defaults to using the original undecorated value, or to using the latest value returned by a decorator that has already ran on the same element.
```

Comment:
Rationale is that it would be surprising if a void returning function would reset a previous decorators new value back to the original value.

I chose to put "undefined" or "void" here because those are terms I thought to search for when looking for this detail, but couldn't find it.